SftpLibsshAdapter/conanfile.py

```diff
@@ -0,0 +1,23 @@
+import os
+from conans import ConanFile
+
+class SFTPLibSSHAdapterConan(ConanFile):
+	name = "SFTPLibSSHAdapter"
+	description = "C++ Wrapper over libssh library to work with sftp"
+	author = "CSW <csw@werfen.com>"
+	topics = ("conan", "sftp")
+	license = "MIT"
+	generators = "visual_studio"
+	settings = "os", "compiler", "build_type", "arch"
+
+	def requirements(self):
+		self.requires("libssh/0.9.5@systelab/stable")
```

Comment:
Don't we need also to include a requirement to the SFTPAdapter interface?

Reply:
It would make sense but is currently a problem if both the interface and the adapter are on the same repository and generate packets altogether.
Anyway it would be advised for the client to specifically target the interface AND the adapter.